generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
@yanan-fu  agree with congli, and some confused when check the test result if someone didn't know the default value.
 (1/2) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.boot_vm_in_hugepage: PASS (955.05 s)